### PR TITLE
Adding topic "Localizing igEditors"

### PR DESCRIPTION
**Pull request by dafo**

### CODE REVIEW

**Comment by dafo** [comment]
This should not be merged as more information must be provided in the new topic.

File: topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+|metadata|
+-->
+
+# How to localize the Editors
+
+There are two types of localization. First is for the localization resources in the controls. Second is for the regional settings in the controls.
+
+Localization resources for the controls are in Bulgarian, Russian, Japanese, German, Spanish and French languages. These reside in js/modules/i18n (where js is the root folder for the JavaScript files in the Ignite UI program installation path). To get more into using the localization resources you can read the ["Customizing the Localization of Ignite UI Controls"](customizing-the-localization-of-netadvantage-for-jquery-controls.html) topic.
+
+The regional settings, on the other hand, provide localized formats including dates, numbers, and currency symbols for the editors. These reside in the ../js/modules/i18n/regional (where js is the root folder for the JavaScript files in the Ignite UI program installation path).
```
Comment (kdinev) [inline topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md]: provide region specific formats for dates, numbers, as well as currency symbols, floating point symbols, decimal separators, default decimal rounding, etc.

File: topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Topic Overview
+
+In this sample we show how a culture can be set to the editors' `regional` option to change the format of dates, times, numbers and currencies. Three regions are configured here (United States, Japan and Tamil, India) and there are many more to choose from in the infragistics.ui.regional-i18n.js files.
```
Comment (kdinev) [inline topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md]: change the default formatting of dates, numbers, and currencies.

**Review by kdinev — CHANGES_REQUESTED** [review]

**Review by kdinev — APPROVED** [review]

**Comment by kdinev** [comment]
@dafo Is this ready for merge?

**Comment by dafo** [comment]
@kdinev I think it is ready to be merged.

**Pull request merged**